src/fetch.ts

```diff
@@ -31,8 +32,21 @@ export function createFetch ({ fetch }: CreateFetchOptions): $Fetch {
       if (opts.params) {
         request = withQuery(request, opts.params)
       }
+      if (opts.body && typeof opts.body === 'object' && ['patch', 'post', 'put'].includes(opts.method?.toLowerCase() || '')) {
```

Comment:
Body can be also of type `FormData` (typeof === object). We should ensure proto is pure object